code/modules/ruins/spaceruin_code/hilbertshotel.dm

```diff
@@ -225,9 +225,9 @@ GLOBAL_VAR_INIT(hhmysteryRoomNumber, 1337)
     var/obj/item/hilbertshotel/parentSphere
 
 /turf/open/space/bluespace/Entered(atom/movable/A)
-    . = ..()
-    A.forceMove(get_turf(parentSphere))
-    do_sparks(3, FALSE, get_turf(A))
+	. = ..()
+	if(A.forceMove(get_turf(parentSphere)))
```

Comment:
`if parentSphere && A.forceMove(blah)`

Reply:
forceMove return 0 if not move, but if you want